lib/after-prepare.js

```diff
@@ -0,0 +1,21 @@
+var fs = require('fs');
+var path = require('path');
+var glob = require('glob');
+var Promise = require('bluebird');
+
+module.exports = function ($logger, $projectData, $usbLiveSyncService) {
+	// delete all scss files from compiled sources
+	var platformsData = $injector.resolve('platformsData');
+	return Promise.each(platformsData.platformsNames, function (platform) {
+		var sassFilesPath = path.join(platformsData.getPlatformData(platform).appDestinationDirectoryPath, 'app/**/*.scss');
+		var sassFiles = glob.sync(sassFilesPath).filter(function (filePath) {
+			var path = filePath;
+			var parts = path.split('/');
+			var filename = parts[parts.length - 1];
+			return path.indexOf("App_Resources") === -1 && filename.indexOf("_") !== 0;
```

Comment:
@manojdcoder Any reason to leave prefixed files in the app packages? `filename.indexOf("_")`

What's the benefit? If none, probably makes sense to remove this check and just remove all .scss files outside of the App_Resources directory.